pkgs/lib.nix

```diff
@@ -1,42 +1,34 @@
 # SPDX-FileCopyrightText: 2023 Jan Tojnar <jtojnar@gmail.com>
 # SPDX-License-Identifier: MIT
+{lib}: {
+  /*
+  * Remove given lines from a string.
```

Comment:
Looks like doc comments are not supported either.

Reply:
Do you suggest another formatter ?

Reply:
There are:
- https://github.com/nix-community/nixpkgs-fmt
- https://github.com/serokell/nixfmt

Reply:
I think we can do without a formatter for now. I still plan to report the issues to Alejandra so that we can use it in future.

Reply:
Actually, I misremembered and documentation comments in Nixpkgs just use a single asterisk: https://github.com/NixOS/nixpkgs/blob/2abea38e0824edd57a3dcfe02f84e09a3a3734c5/lib/attrsets.nix#L888